examples/jackd-talker/jackd_talker.c

```diff
@@ -766,7 +766,7 @@ int pci_connect()
 		}
 		printf("attaching to %s\n", devpath);
 		err = igb_attach(devpath, &igb_dev);
-		if (err) {
+		if (err || igb_attach_tx( &igb_dev )) {
```

Comment:
This logic looks confusing. Why wouldn't the error from igb_attach_tx() be handled more clearly?